common/types/primitive/hashes.go

```diff
@@ -0,0 +1,193 @@
+package primitive
```

Comment:
The reason this package exists is because otherwise we would have a cyclic dependency between `types` and `wire`

In my oppinion the fix for this is to move the types in `wire` to the `activation` package and allow them to import `types` but `types` must not depend on `activation`. That's why converters from `ActivatonTx` to `wire.V1` cannot be methods on the `ActivationTx` type but have to be functions in the `activation` package (and should also only be used there).